tools/ds-identify

```diff
@@ -281,6 +280,50 @@ read_fs_info() {
     DI_ISO9660_DEVS="${isodevs#,}"
 }
 
+read_geom() {
+    local oifs="$IFS" line="" delim=","
+    local ret=0 out="" labels="" dev="" label="" ftype="" isodevs=""
+    out=$(geom label status -as) || {
```

Comment:
after adding the 'geom_label_status_as' function, then you'll just use the variable, so here this would look like:

    geom_label_status_as
    local out="$DI_GEOM_LABEL_STATUS_OUT"
    ...
    [ "$out" ] = "$UNAVAILABLE" && return 0
    
    and the rest of your function here.

Reply:
you mean
```sh
[ "$out" = "$UNAVAILABLE" ] && return 0
```
right?

Reply:
yes. good lint'ing.

Reply:
hmmmm…
for now i'm passing on the return value instead of return 0… not sure that's a sensible thing to do